src/main/java/blossom/task/Event.java

```diff
@@ -0,0 +1,25 @@
+package blossom.task;
+
+public class Event extends Task {
+    private String from;
+    private String to;
```

Comment:
Maybe using a more descriptive name such as startTime and endTime might be better here.